aci/resource_aci_vnsabsnode.go

```diff
@@ -539,9 +546,19 @@ func resourceAciFunctionNodeUpdate(ctx context.Context, d *schema.ResourceData,
 		return diag.FromErr(err)
 	}
 
+	vnsAbsFuncConnAttr := models.FunctionConnectorAttributes{}
+	vnsAbsFuncConnAttr.Annotation = "{}"
+	if vnsAbsNodeAttr.IsCopy == "yes" {
```

Comment:
Are we allowed to toggle isCopy from "yes" to "no"? What will be the update behaviour for this scenario? Do we need to remove objects?

Reply:
it can't be changed from yes->no 
it gives the following error
Validation failed: isCopy property of existing AbsNode cannot be changed for a COPY node

Reply:
Is this error from apic? If not should we detect during plan?

Reply:
Also, this is changed to forceNew. so, now the resource will be deleted and recreated.